src/MenuItem/ItemStyleInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace PhpSchool\CliMenu\MenuItem;
+
+use PhpSchool\CliMenu\Style;
+
+interface ItemStyleInterface
```

Comment:
Can we rename this `StyleableItem`

Reply:
`StyleableItem` or `StyleableItemInterface`?

Reply:
I think we can just go with `StyleableItem`

Reply:
Thinking about it again, we also don't need this. I think we can drop it. Each item can still implement `getStyle` and `setStyle` but should type hint on the concrete style they need. If it turns out that a few different types of item use basically the same styles then we can introduce an interface but I think until then there is no benefit.